src/competencies/management/commands/export_competencies.py

```
@@ -0,0 +1,32 @@
+
+    def handle(self, *args, **options):
+        if options['format'] == 'csv':
+            content = export_csv()
+        else:
+            content = json.dumps(export_competency_set(), indent=4) + '\n'
+
+        if options['output']:
+            try:
+                Path(options['output']).write_text(content, encoding='utf-8')
+            except OSError as e:
+                raise CommandError(str(e))
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)